tests/test_instalabel.py

```diff
@@ -0,0 +1,50 @@
+import shippo
+from shippo.models.components import Carriers, DistanceUnitTemplate, WeightUnit, \
+    AddressCreateRequest, ShipmentCreateRequest, InstantTransactionRequestBody, \
+    ParcelCreateRequest
+from tests.helpers_custom import get_carrier_account
+
+
+class TestInstalabel:
+
+    def test_instalabel(self, api: shippo.Shippo):
+        carrier_account = get_carrier_account(api, Carriers.USPS)
+
+        create_transaction_response = api.transactions.create_transaction(
+            request_body=InstantTransactionRequestBody(
+                carrier_account=carrier_account.object_id,
+                servicelevel_token="usps_ground_advantage",
+                shipment=ShipmentCreateRequest(
+                    address_from=AddressCreateRequest(
+                        name="Rachael",
+                        street1="1092 Indian Summer Ct",
+                        city="San Jose",
+                        state="CA",
+                        zip="95122",
+                        country="US",
+                        phone="4159876543",
+                        email="rachael@alltheyarnz.com"
+                    ),
+                    address_to=AddressCreateRequest(
+                        name="Mr Hippo",
+                        street1="965 Mission St #572",
+                        city="San Francisco",
+                        state="CA",
+                        zip="94103",
+                        country="US",
+                        phone="4151234567",
+                        email="mrhippo@shippo.com"
+                    ),
+                    parcels=[
+                        ParcelCreateRequest(
+                            length="5",
+                            width="5",
+                            height="5",
+                            distance_unit=DistanceUnitTemplate.CM,
+                            weight="2",
+                            mass_unit=WeightUnit.LB
+                        )
+                    ]
+                )
+            ))
+        assert create_transaction_response.transaction is not None
```

Comment:
why not assert anything about the response?

Reply:
b/c these tests are not meant to validate the endpoints, more about SDK usability and providing language-specific examples of major user flows.  this is effectively asserting that the response code is 201 and the transaction was successfully created